WordPress/Classes/Categories/NSString+Helpers.m

```diff
@@ -188,4 +188,13 @@ - (NSArray *)tokenize
     return tokens;
 }
 
+- (BOOL)isWordPressComURL
+{
+    NSRegularExpression *protocol = [NSRegularExpression regularExpressionWithPattern:@"wordpress\\.com/?$" options:NSRegularExpressionCaseInsensitive error:nil];
```

Comment:
Should we do the static var, dispatch_once optimization dance that we do with other NSRegularExpressions?  Maybe we don't need it now but it might be nice if this method gets called frequently in the future.